### PR TITLE
ENH: ReaderWithFilestore now uses its own read

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: bluesky/examples.py [hunk 1]
```diff
@@ -523,7 +523,7 @@ def trigger(self):
         return NullStatus()
 
     def read(self):
-        return self._result
+        return super().read()
```
Comment (tacaswell) [inline bluesky/examples.py]: Just remove this method.

Comment (CJ-Wright) [inline bluesky/examples.py]: That causes read to give back arrays not filestore uids.

Comment (danielballan) [inline bluesky/examples.py]: @CJ-Wright What? I'm not sure how this is different than just removing the method.

**Comment by danielballan** [comment]
bump

**Comment by CJ-Wright** [comment]
Yea, sorry that I haven't fixed this yet (although I guess this might be of interest with the configuration refactor?)

**Comment by CJ-Wright** [comment]
I'm superseding this with #698

**Pull request closed without merging**